src/types/client.d.ts

```diff
@@ -39,7 +39,7 @@ declare class Client extends GeneratedApiClient {
   clientId: string;
   scopes: string[];
   privateKey: string;
-  oauth: Oauth;
+  oauth: OAuth;
   http: Http;
 }
 
```

Comment:
Would be better to use `export declare class Client` (named export) vs. `export default Client` (default export).

The default export is very inflexible, we will definitely want to export more than one symbol from a single file (this one also has ConfigProperties, where does that get exported?)  `export interface ConfigProperties`

Reply:
Done.

`ConfigProperties` looks redundant - replaced with object type literal.